opent2t-reference/Opent2tHelper.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+var q = require('q');
```

Comment:
Should this be converged with the helpers  used by teh CLI? Lots of such 'proxies' now in our code base -- maybe time to consolidate? I found myself writing one as well for the proximal case temporarily but those ended up still using the TranslatorCli/OnboardingCli helpers in the CLI